tests/Integration/Behaviour/Features/Context/Domain/Product/SpecificPriceContext.php

```diff
@@ -278,6 +301,91 @@ private function createAddSpecificPriceCommand(int $productId, TableNode $tableN
         return $addCommand;
     }
 
+    /**
+     * @param int $specificPriceId
+     * @param TableNode $tableNode
+     *
+     * @return EditProductSpecificPriceCommand
+     */
+    private function createEditSpecificPriceCommand(int $specificPriceId, TableNode $tableNode): EditProductSpecificPriceCommand
+    {
+        $dataRows = $tableNode->getRowsHash();
+        $editCommand = new EditProductSpecificPriceCommand($specificPriceId);
+
+        if (isset($dataRows['reduction type'], $dataRows['reduction value'])) {
+            $editCommand->setReduction($dataRows['reduction type'], (string) $dataRows['reduction value']);
+        }
+        if (isset($dataRows['includes tax'])) {
+            $editCommand->setIncludesTax(PrimitiveUtils::castStringBooleanIntoBoolean($dataRows['includes tax']));
+        }
+        if (isset($dataRows['price'])) {
+            $editCommand->setPrice($dataRows['price']);
+        }
+        if (isset($dataRows['from quantity'])) {
+            $editCommand->setFromQuantity((int) $dataRows['from quantity']);
+        }
+        if (isset($dataRows['combination'])) {
+            $editCommand->setCombinationId($this->getStoredId($dataRows, 'combination'));
+        }
+        if (isset($dataRows['shop group'])) {
```

Comment:
I believe this one has been removed, it's surprising that neither CS-fixer nor PHPStan screams about this